lms/templates/course_modes/track_selection.html

```diff
@@ -0,0 +1,193 @@
+                        <div class="certificate-container">
+                            <div class="triangle-overlay"></div>
+                            <span class="trophy-icon">
+                                <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path d="M26.6667 3.33333H23.3333V0H6.66667V3.33333H3.33333C1.5 3.33333 0 4.83333 0 6.66667V8.33333C0 12.5833 3.2 16.05 7.31667 16.5667C8.36667 19.0667 10.6167 20.95 13.3333 21.5V26.6667H6.66667V30H23.3333V26.6667H16.6667V21.5C19.3833 20.95 21.6333 19.0667 22.6833 16.5667C26.8 16.05 30 12.5833 30 8.33333V6.66667C30 4.83333 28.5 3.33333 26.6667 3.33333ZM3.33333 8.33333V6.66667H6.66667V13.0333C4.73333 12.3333 3.33333 10.5 3.33333 8.33333ZM26.6667 8.33333C26.6667 10.5 25.2667 12.3333 23.3333 13.0333V6.66667H26.6667V8.33333Z" fill="#EFF8FA"/>
+                                </svg>
+                            </span>
+                            <p class="price-display">${currency_symbol}${min_price} ${currency}</p>
+                            <div class="choice-title"><h4>${_("Earn a certificate")}</h4></div>
+                            <div class="choice-bullets">
+                                <ul>
+                                    <li>Showcase a <a class="verified" href="https://www.edx.org/verified-certificate" target="_blank"><b><u> verified certificate</u></b></a> of completion on your resumé to advance your career</li>
+                                    <li>Get <b>access to all course activities</b>, including both graded and non-graded assignments, while the course is running</li>
+                                    <div class="collapsible">Show more</div>
+                                    <li class="collapsible-item"><b>Gain full access</b> to course content and materials, even after the course ends <span class="popover-icon"><i class="fa fa-question-circle" aria-hidden="true"></i><span class="popover"><a href="https://support.edx.org/hc/en-us/articles/360013426573-What-are-the-differences-between-audit-free-and-verified-paid-courses-" target="_blank"><u>Learn more</u></a> about course access</span></span></li>
+                                    <li class="collapsible-item">Support our <b>non-profit mission</b> to increase access to high-quality education for everyone, everywhere</li>
+                                </ul>
+                            </div>
+                            <!-- JK TODO: check ids, do a separate template? -->
+                            <ul class="list-actions">
+                            <li class="action action-select upgrade-button">
+                                <input type="hidden" name="contribution" value="${price_before_discount or min_price}" />
+                                    <button id="track_selection_upgrade" type="submit" name="verified_mode">
```

Comment:
Do we want to keep the same ID as the current Track Selection upgrade button in terms of tracking the event? I could have 1) a separate ID 2) same ID but different pageName for tracking.

Reply:
Good question! I'd think we can keep it the same- (for the other VP pages we've just replaced them, and we haven't differentiated between versions in the past, or happy path vs non)- but I'll defer to Seth :)

Reply:
Commenting here to close the loop - discussed during standup, will keep the same ID for tracking since we'll roll it out to all users after UX testing, so we can use the date to know where the new metrics for the new page are in + like Diane mentioned we've kept the IDs the same for other pages.